core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -267,20 +261,22 @@ fun LpOutlinedTextFieldInput(
  * @param title refers to the label that the text-field will have
  * @param items refers to the elements of the DropDownList (list of elements)
  * @param selected refers to a high order function that returns the selected option of the DropDownList
+ * @param value The current value of the input field
  *
  * @author Jesus Lopez Gonzalez
  * @modifiedBy Jose Miguel Garcia Reyes
+ * @modifiedBy Irving Gonzalez
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun DropdownTextField(
```

Comment:
Please test if it is working

Reply:
This component  works:

<img width="940" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/a10243b8-cad9-473f-804d-16c0dc64121e">